src/core/mac/mac_frame.cpp

```diff
@@ -275,6 +277,20 @@ void Frame::SetFramePending(bool aFramePending)
     }
 }
 
+void Frame::SetIePresent(bool aIePresent)
+{
+    uint16_t fcf = GetFrameControlField();
+
+    if (aIePresent)
+    {
+        SetFrameControlField(fcf | kFcfIePresent);
+    }
+    else
+    {
+        SetFrameControlField(fcf & ~kFcfIePresent);
```

Comment:
nit: does it save any code space to simply modify `fcf` first and have a single `SetFrameControlField()` call at the end?

Reply:
Good point! 
Changed this in new pushed commit.